api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -181,6 +181,12 @@
      */
     Attribute<Class<?>> TUPLE_CLASS = new Attribute<>(Class2String.getInstance());
 
+    /**
```

Comment:
It's in PortContext not DAGContext. It can stay as a placeholder to indicate that the plan is to go stream level. I can add a TODO or I can remove it. Let me know.